Draft ds-sig charter (#193)
operate-first/community: sig-data-science/charter.md

# SIG Data Science Charter

This charter adheres to the conventions described in the [Operate First Governance document](https://github.com/operate-first/community/blob/main/governance.md) and uses the Roles and Organization Management outlined in that document.

## Scope

> SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First Community. This includes the development, exploration and communication of  cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the open services group.

Reviewer: For this suggestion, I'm also splitting the paragraph into a sentence per line to make the diff easier to read:

```suggestion
SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First community.
This includes the development, exploration, and communication of cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the [Open Services Group](https://github.com/open-services-group/community/).
```

### In scope

#### Code, Binaries and Services

* Publish and promotion of videos, blogs, talks and tutorials on the tools and best practices developed by sub-projects.

> * Support the onboarding of new data science projects.

Reviewer: ```suggestion
* Support the onboarding of new Open Source data science projects.
```

* Experiment with new cloud native data science development tools and methods.
* Leverage the data produced by the operate first community cluster to provide AIOps tools and analyses to the community.

#### Cross-cutting and Externally Facing Processes

* Provide guidance to external groups with onboarding and development of data science work on any of the Operate First managed clusters.
* Foster a two-way relationship between Operate First and the wider open source data science community at large.
* Work with other Operate First SIG’s to ensure data collection and reporting is ongoing.

### Out of scope

* Not responsible for the installation and availability of data science services maintained by the Operate First Community cloud.

> * Not responsible for any code, notebooks or projects outside of the Operate First Community, and without an associated sub-project initiated by this SIG.

Reviewer: ```suggestion
* Not responsible for any code, notebooks, or projects outside of the Operate First community, and without an associated subproject initiated by this SIG.
```

> * Not responsible for any request for support on any non-open source data or code.

Reviewer: ```suggestion
* Not responsible for any request for support on any non-Open Source data or code.
```

## Roles and Organization Management

> This sig adheres to the Roles and Organization Management outlined in [governance document](https://github.com/operate-first/community/blob/main/governance.md)

Reviewer: ```suggestion
This SIG adheres to the Roles and Organization Management outlined in [governance document](https://github.com/operate-first/community/blob/main/governance.md)
```
